.gitignore

```diff
@@ -57,3 +57,6 @@ typings/
 # dotenv environment variables file
 .env
 
+# never commit lock files to keep testing working
+yarn.lock
+package-lock.json
```

Comment:
why is this needed again?

Reply:
This is because of this failure: https://travis-ci.com/ember-cli/create-ember-app/jobs/167640273#L509

Essentially if you have a yarn.lock or a package-lock.json it will fail because the `npm install` or `yarn install` steps will be installing the locked version and not what is defined in the package.json (currently set to `"ember-cli": "^3.5.1"`) and therefore would not install the latest version

Reply:
hmm... okay...